libs/blog/pages/feature-page/src/lib/page-details-container/page-details-container.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, computed, inject, input } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+import { PageDetailsStore } from '@angular-love/blog/pages/data-access';
+import { UiPageContentComponent } from '@angular-love/blog/pages/ui-page-content';
+
+import { PageDetailsSkeletonComponent } from '../page-details/page-details-skeleton.component';
+
+@Component({
+  selector: 'al-page-details-container',
+  standalone: true,
+  imports: [UiPageContentComponent, PageDetailsSkeletonComponent],
+  templateUrl: './page-details-container.component.html',
+})
+export class PageDetailsContainerComponent {
+  private readonly pageDetailsStore = inject(PageDetailsStore);
+  private readonly _domSanitizer = inject(DomSanitizer);
```

Comment:
private props should go below public ones

Reply:
I need them above for property initialization I use below